python_modules/dagster/dagster/_core/definitions/freshness_checks/last_update.py

```diff
@@ -50,6 +50,16 @@ def build_last_update_freshness_checks(
     parameter to "3 hours". This would mean that the check will expect the most recent observation
     record to indicate data no older than 3 hours, relative to the current time, regardless of when it runs.
 
+    The check result will contain the following metadata:
+    "dagster/freshness_params": A dictionary containing the parameters used to construct the
+    check
+    "dagster/last_updated_time": The time of the most recent update to the asset
+    "dagster/overdue_seconds": (Only present if asset is overdue) The number of seconds that the
+    asset is overdue by.
+    "dagster/overdue_deadline_timestamp": The timestamp that we are expecting the asset to have
+    arrived by. In the case of a provided deadline_cron, this is the timestamp of the most recent
+    tick of the cron schedule. In the case of no deadline_cron, this is the current time.
```

Comment:
I'll share this feedback with the rest of my checks dogfooding notes, but I think this definition of deadline is confusing. We can continue with the current names for now though

Reply:
i agree that it's confusing. This is about the best I could come up with for now; but open to being wrong.